.circleci/config.yml

```diff
@@ -188,6 +189,7 @@ jobs:
           docker_layer_caching: true
       - run:
           name: Test
+          no_output_timeout: 30m
           command: cargo test --all --features docker --jobs 4 -- --test-threads 4
```

Comment:
while we're at it we can probably remove the thread limit here.

Reply:
Sounds good to me, as `large` executors have 4 cores too, so this might be redundant.